src/applications/personalization/rated-disabilities/components/TotalRatedDisabilities.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import PercetnageCalloutBox from '../components/PercentageCalloutBox';
+
+const TotalRatedDisabilities = props => {
+  const { loading, totalDisabilityRating, error } = props;
+  let content;
+  // If the data from the parent is loading ( loading prop ), show a loading indicator
+  // If there is an error, display an error message,
+  // If there is no rating, display a no rating message
+  // If there is a rating, display the rating and content
+  if (loading) {
+    content = (
+      <LoadingIndicator message="Loading your total disability rating..." />
+    );
+  } else if (error) {
+    const message = (
+      <span>
+        <p>
+          We're sorry. An error occurred when accessing your disability rating
+          information.
+        </p>
+        <h4>What you can do</h4>
+        <p>
+          Sign out of VA.gov, then log back in to try this page again. If the
+          error continues, please call the VA.gov Help Desk at 1-855-574-7286
+          (TTY:1-800-829-4833). We're here Monday-Friday, 8:00 a.m. - 8:00 p.m.
+          (ET).
+        </p>
+      </span>
+    );
+    content = (
+      <AlertBox
+        headline="Total disabilities error"
+        content={message}
+        status="error"
+        isVisible
+      />
```

Comment:
Theoretically there will be two different error states the frontend might receive:
1. 500 error - handled above.
2. 401 error - no rating was found on file. 

I created https://github.com/department-of-veterans-affairs/va.gov-team/issues/2726 to add an additional error case here because the backend work for total combined rating has yet to be finished. That ticket will also house the work for updating our error messaging in the `RatedDisabilityList` component.

Reply:
@micahchiang The code I have can handle both 500 and 401 errors. There is a different error shown if no total rating was found but there was not a 500 error.

Reply:
My bad! Didn't look below.